code/modules/client/preferences.dm

```diff
@@ -152,9 +157,8 @@ datum/preferences
 	var/savefile/loaded_character
 	var/datum/category_collection/player_setup_collection/player_setup
 
-	var/dress_mob = TRUE
-
-
+	var/icon/bgstate = "000"
```

Comment:
Isn't this default state an icon_state not an icon?

Reply:
lol you're right